conf/monitorix.conf

```
@@ -586,8 +586,12 @@ secure_log_date_format = %b %e
 <fail2ban>
         list = Security, Overload / Abuse
```

The `list` configuration should have the same number of items as the `desc` list
```suggestion
        list = Overload / Abuse, YunoHost, Mail, Default, SSH, Apps
```